index.bs

```diff
@@ -257,7 +257,8 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 <h4 id="trace" oldids="trace-data,dom-console-trace" method for="console">trace(...<var>data</var>)</h4>
 
-Perform Logger("error", <var>data</var>).
+Performs Logger("log", <var>data</var>) and prints a snapshot of the call-stack from where the method was called.
```

Comment:
Well, maybe not "text" produced by the %o and %O. The idea is they could produce interactive widgets (and they do in some implementations)